code/xcvm/cosmwasm/contracts/interpreter/Cargo.toml

```diff
@@ -0,0 +1,37 @@
+[package]
+authors = ["Composable Developers"]
+description = "XCVM interpreter contract"
+edition = "2021"
+license = "Apache-2.0"
+name = "xcvm-interpreter"
+repository = "https://github.com/ComposableFi/composable"
+version = "0.1.0"
+
+exclude = [
+  # Those files are rust-optimizer artifacts. You might want to commit them for convenience but they should not be part of the source code publication.
+  "xcvm-interpreter.wasm",
+  "hash.txt",
+]
+
+[lib]
+crate-type = ["cdylib", "rlib"]
+
+[features]
+# use library feature to disable all instantiate/execute/query exports
+library = []
+
+[dependencies]
+cosmwasm-std = { version = "1.0.0" }
+cw-storage-plus = "0.13.2"
+cw20 = "0.13.2"
+num = "0.4"
+schemars = "0.8.8"
+serde = { version = "1.0.137", default-features = false, features = ["derive"] }
+serde-json-wasm = { git = "https://github.com/hussein-aitlahcen/serde-json-wasm", rev = "1608a13d2a2ba90605d9626a51ff6667aca5a2d6" }
```

Comment:
not sure. as per spec protobuf and json formats has no dictionaries. that is only view side interpretation. in proto files maps are not really maps too. 

so could just do serde interpretation of maps on client side with out fork.

I just want to say that adding map to low level embeeded json likely will not merge map interpretation into core. really protobuf and json have only vectors of tuples - bags, not sets.

Reply:
I haven't use it but seen this: https://developers.google.com/protocol-buffers/docs/proto#maps.

Reply:
We talked to @hussein-aitlahcen and it is doable if your key is a simple type like string or int. So in our case, we can use it without a problem.

Reply:
language has map. but if you will send several same keys it still will be up to spec. so they tell have map. but there is not restriction to have map on message format. example, such messages likely will fail evm parser if it linear (not sure it is)